CHANGELOG.released.md

```diff
@@ -7,6 +7,32 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MICRO`.
 For upgrade instructions, please check the [migration guide](MIGRATIONS.released.md).
 
+## [20.12.0](https://github.com/scalableminds/webknossos/releases/tag/20.12.0) - 2020-11-23
+[Commits](https://github.com/scalableminds/webknossos/compare/20.11.0...20.12.0)
+
+### Highlights
+- Added multi-resolution volume annotations. As a result, volume tracings can be viewed even when zooming out to downsampled magnifications. Note that already existing volume tracings will still only contain data in the first magnification. If you want to migrate an old volume tracing, you can download and re-import it. [#4755](https://github.com/scalableminds/webknossos/pull/4755)
```

Comment:
download and re-import is no longer correct, I think. There should be an explicit downsampling dialogue instead :)

Reply:
Good catch!